plugins/view-serviceaccount-kubeconfig.yaml

```diff
@@ -14,8 +14,18 @@ spec:
       matchLabels:
         os: darwin
         arch: amd64
-  - uri: https://github.com/superbrothers/kubectl-view-serviceaccount-kubeconfig-plugin/releases/download/v2.2.5/kubectl-view_serviceaccount_kubeconfig-linux-amd64.zip
-    sha256: 3c50abc71aff4346775d76a6090e0ea6e0189a23fcf4d0d95c83bfc84f4b83de
+  - uri: https://github.com/superbrothers/kubectl-view-serviceaccount-kubeconfig-plugin/releases/download/v2.2.6/kubectl-view_serviceaccount_kubeconfig-darwin-arm64.zip
+    sha256: d68510c91675fb9ee3054f6ac08be3e2cf22d24df3e2de2724a07caa88fa96d9
+    bin: kubectl-view_serviceaccount_kubeconfig
+    files:
+    - from: "*"
+      to: .
```

Comment:
This is the default behavior. Would you care to strip those File specs?

Reply:
Thank you for your suggestion. I've updated here as you say.